app/src/pages/settings/NewUserDialog.tsx

```diff
@@ -82,22 +81,31 @@ export function NewUserDialog({
   );
 
   return (
-    <DialogContainer
-      onDismiss={onDismiss}
+    <Modal
+      isOpen={true}
+      onOpenChange={(isOpen) => {
+        if (!isOpen) {
+          onDismiss();
+        }
+      }}
       isDismissable
-      type="modal"
-      isKeyboardDismissDisabled
     >
-      <Dialog title="Add user">
+      <Dialog>
+        <h2>Add User</h2>
```

Comment:
This is styled very badly and inconsistently. In
<img width="811" alt="Screenshot 2025-05-29 at 7 14 23 PM" src="https://github.com/user-attachments/assets/f1dbd3fc-d095-450e-ab06-a488961b66d2" />

In addition there is no longer a cancel button to exit this. Can you at least make this match the other new modals